index.html

```diff
@@ -62,28 +62,40 @@ <h1>Hey there.<br />I'm Mesfin</h1>
+              </svg>
+              </a>
         </li>
         <li>
           <a href="https://www.linkedin.com">
-            <img src="icons/Linkedin.svg" alt="linkedin addres" />
+            <svg width="20" height="20" viewBox="0 0 20 20" fill=""  xmlns="http://www.w3.org/2000/svg">
+              <path class="icons" d="M13.6444 3.03467C10.1556 3.03467 7.28857 5.86225 7.28857 9.3795V19.3105C7.28857 19.6898 7.59946 20.0002 7.97942 20.0002H11.5718C11.9518 20.0002 12.2627 19.6898 12.2627 19.3105V9.65536C12.2627 8.89674 12.8844 8.27605 13.6444 8.27605C14.4043 8.27605 15.0261 8.89674 15.0261 9.65536V19.3105C15.0261 19.6898 15.3369 20.0002 15.7169 20.0002H19.3093C19.6893 20.0002 20.0001 19.6898 20.0001 19.3105V9.3795C20.0001 5.89674 17.1677 3.03467 13.6444 3.03467Z" fill="#CFCFCF"/>
+              <path class="icons" d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z" fill="#CFCFCF"/>
+              <path class="icons" d="M4.28325 6.27588H0.690846C0.310881 6.27588 0 6.58622 0 6.96553V19.3104C0 19.6897 0.310881 20 0.690846 20H4.28325C4.66321 20 4.97409 19.6897 4.97409 19.3104V6.96553C4.97409 6.58622 4.66321 6.27588 4.28325 6.27588Z" fill="#CFCFCF"/>
+              </svg>              
           </a>
         </li>
         <li>
           <a href="https://twitter.com">
-            <img src="icons/twitter.png" alt="twitter address"
-          /></a>
+            <svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path class="icons" d="M20.1753 0.394423C19.842 -0.00424693 19.2753 -0.103914 18.842 0.195088L16.9086 1.52399C16.0086 0.560536 14.742 0.0289756 13.3753 0.0289756C10.6753 0.0289756 8.44198 2.22166 8.44198 4.94591C8.44198 5.34458 8.47532 5.71002 8.57532 6.07547C6.24198 5.57713 4.84198 3.88279 4.30865 0.826316C4.24198 0.427646 3.94198 0.128643 3.54198 0.0289756C3.17532 -0.070692 2.77532 0.0954206 2.54198 0.394423C2.47532 0.460868 -3.02468 8.10205 4.94198 14.5804C4.17532 16.2083 1.87532 17.6037 0.975315 18.0688C0.541982 18.2681 0.341982 18.7332 0.441982 19.1984C0.541982 19.6635 0.941982 19.9625 1.40865 19.9625H3.14198C13.2753 19.9625 17.8753 14.9459 17.9753 3.71667L20.0753 1.72332C20.4753 1.3911 20.5086 0.793093 20.1753 0.394423Z" fill="#505F79"/>
+              </svg>
+              </a>
```

Comment:
- [Optional] Kindly consider fixing your mobile version to be responsive and look like the figma template. You are expected to stick to the design as much as possible.
### Your design
![mm](https://user-images.githubusercontent.com/94551484/193318267-9d1c6a4f-7827-4a08-8b12-20dcddf453be.png)

### Figma template
![mm1](https://user-images.githubusercontent.com/94551484/193318317-257c9662-58e2-44fc-b55f-df4142d3b195.png)

- [Optional] Please move this SVG image to your CSS file. It is expected that every image strictly for decoration purposes should be added as `background-image` in CSS file. This is best practice.